hugegraph-llm/src/hugegraph_llm/operators/llm_op/schema_build.py

```diff
@@ -0,0 +1,128 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+import json
+import re
+from typing import List, Dict, Any, Optional
+
+from hugegraph_llm.models.llms.base import BaseLLM
+from hugegraph_llm.models.llms.init_llm import LLMs
+from hugegraph_llm.utils.log import log
+
+
+class SchemaBuilder:
+    """Automated Schema Generator"""
+
+    def __init__(
+        self,
+        llm: Optional[BaseLLM] = None,
+        schema_prompt: Optional[str] = None,
+    ):
+        self.llm = llm or LLMs().get_chat_llm()
+        self.schema_prompt = schema_prompt or (
+            "You are a Graph Schema generator. Based on the following three parts of content, "
+            "output a Schema JSON that complies with HugeGraph specifications:\n\n"
+            "1. Raw data samples:\n{raw_texts}\n\n"
+            "2. Query examples (description + Gremlin):\n{query_examples}\n\n"
+            "3. Few-Shot Schema examples:\n{few_shot_schema}\n\n"
+            "Please return only the JSON object, without any additional explanations."
+        )
```

Comment:
separate the prompt str & use `"""str"""` for it

better use a basic format for it